test/v200/utils/common/component_test_utils.go

```diff
@@ -43,6 +43,12 @@ func (devfile *TestDevfile) AddComponent(componentType schema.ComponentType) sch
 	if componentType == schema.ContainerComponentType {
 		component = devfile.createContainerComponent()
 		devfile.SetContainerComponentValues(&component)
+	} else if componentType == schema.KubernetesComponentType {
+		component = devfile.createKubernetesComponent()
```

Comment:
I'm thinking perhaps we don't need to split those component creation into sub function calls. 
Those function call are almost identical, except for the part of creating an empty component. 
can combine those sub function calls within this `AddComponent`

```suggestion
		func (devfile *TestDevfile) AddComponent(componentType schema.ComponentType) schema.Component {

	LogInfoMessage("Create a %v component :", componentType)
	component := schema.Component{}
	component.Name = GetRandomUniqueString(8, true)
	LogInfoMessage(fmt.Sprintf("....... Name: %s", component.Name))
	switch componentType{
	case schema.ContainerComponentType:
	      component.Container = &schema.ContainerComponent{}
	      devfile.SetContainerComponentValues(&component)
	......   // cases for other component types
	}
	devfile.componentAdded(component)
	return component
}
```

Same for `AddCommand`

Reply:
@kim-tsao I guess I'm late for this PR..

Reply:
@yangcao77 sorry I missed this!  I appreciate your feedback anyhow so I'll try to apply your suggestion next time, thanks.